File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.py
def solution(n, control):
    con_list = list(control)
    
    for i in con_list:
        if i == 'w':
            n += 1
        elif i == 's':
            n -= 1
        elif i == "d":
            n += 10
        elif i == 'a':
            n -= 10
    return n

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.java
class Solution {
    public int solution(int n, String control) {
         for (int i = 0; i < control.length(); i++) {
              char command = control.charAt(i);
            switch (command) {
                case 'w':
                    n += 1;
                    break;
                case 's':
                    n -= 1;
                    break;
                case 'd':
                    n += 10;
                    break;
                case 'a':
                    n -= 10;
                    break;
                default:
                    break;
            }
        }
        return n;
    }
    
}